### DISCUSSION TITLE
Chamilo and LTI Advantage - no interface available

### DISCUSSION THREAD

**Discussion by SP-CMQ** [chamilo/chamilo-lms#5821] [General]
I am currently on version 1.11.26, but the page `plugin/ims_lti/admin.php` always return an error 500 stating "unavailable" (we check on the server and the page is there and is conform with the vanilla version). We don't know exactly since when this page is not responding correctly, but we have another server that is still on 1.11.20 that have the exact same problem. We didn't notice since when this problem occurs as we don't have to change our LTI setup often, but now that we want to add a new one, we don't have access to the interface that generate the client id (in worst case scenario, if we are able to get access to that ID by other means temporarily, we could set it manually in the database, but a more stable solution would be ideal.)

EDIT1: we are still able to add new LTI tools by lauching `plugin/ims_lti/create.php` and everything seems to work fine in the creation of the LTI. We also don't have that problem if we add the LTI from the course settings instead of the admin.

**Comment by ywarnier** [comment]
Hi @SP-CMQ 
Error 500 usually indicates an error on the server side. Do you have access to the Apache error_log (if you use Apache)? It should show an error message that would help us help you.

**Reply by SP-CMQ** [reply]
It has something to do with our database... We uninstall LTI plugin and reinstall it and it bring it back on, but restoring the data in the table broke it again. We are investigating to find the corrupted data. Thanks still.
